src/Agent.Plugins/PipelineArtifact/PipelineArtifactPlugin.cs

```diff
@@ -13,21 +13,27 @@
 using Microsoft.TeamFoundation.DistributedTask.WebApi;
 using Microsoft.VisualStudio.Services.Agent.Util;
 using Agent.Sdk;
+using System.Text.RegularExpressions;
 
 namespace Agent.Plugins.PipelineArtifact
 {
     public abstract class PipelineArtifactTaskPluginBase : IAgentTaskPlugin
     {
         public abstract Guid Id { get; }
-        public string Version => "0.139.0"; // Publish and Download tasks will be always on the same version.
+        public string Version => "0.140.0"; // Publish and Download tasks will be always on the same version.
         public string Stage => "main";
 
         public async Task RunAsync(AgentTaskPluginExecutionContext context, CancellationToken token)
         {
             ArgUtil.NotNull(context, nameof(context));
-
             // Artifact Name
-            string artifactName = context.GetInput(ArtifactEventProperties.ArtifactName, required: true);
+            string artifactName = context.GetInput(ArtifactEventProperties.ArtifactName, required: false);
+
```

Comment:
This code, in my understanding will break the download task as it will insert an artifact name that doesn't exist